### ISSUE TITLE
[Accessibility] Navigating back to previous location in infinitely scrolling page

### ISSUE THREAD

**Issue by abhipanda** [PeachScript/vue-infinite-loading#144]
I am coming from a more accessibility point of view, how can I navigate back to the previous location when I clicked on a link and came back. Ideally the list item should be highlighted shortly something like https://try.discourse.org/

**Comment by PeachScript** [comment]
It is outside the scope of this component, perhaps you can use `keep-alive` feature of `Vue.js` and `scrollBehavior` of `vue-router` to implement it.
